pkg/util/sys/log.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sys
+
+import "github.com/coreos/pkg/capnslog"
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "util/sys")
```

Comment:
I actually have no problem with having a new file that just contains the logger init.  I originally wanted to add logger init into existing files, but in a lot of cases there was no intuitive place to do so.  I'm OK with having a log.go file in each package, it keeps logic nicely partitioned within a package.